src/engine/controls/bpmcontrol.cpp

```diff
@@ -803,14 +803,29 @@ double BpmControl::getBeatMatchPosition(
     double thisDivSec = (dThisNextBeat - dThisPosition)
             / dThisSampleRate / dThisRateRatio;
 
+    if (dOtherBeatFraction < 1.0/8) {
```

Comment:
I was thinking an eighth note, not an eighth of a beat (which is a quarter note).  An eighth note would be 50% of a beat, which we decided is too long, but I think this is too quick.  Can you make it 1/4 instead?

Reply:
I am thinking about it musically, and a sixteenth note feels like a good amount of time to me.  The current value feels a little too strict